src/main/resources/resources/logisim/strings/fpga/fpga_pl.properties

```diff
@@ -2,8 +2,9 @@
 # Reviewed MOR: 2021-07-05 (WiP)
 #
 FPGANotSupported = Brak wsparcia
-FPGASupported = Wspierane
+FPGASupported = Obs?ugiwane
```

Comment:
Is the question mark `?` correct in between?

Reply:
No, there should be accented character. For some odd reasons this file was taken as ISO-8859-1 encoded instead of UTF8. Fixed